src/java.base/unix/native/libjava/nio/fs/UnixNativeDispatcher.c

```diff
@@ -218,7 +218,9 @@ static unlinkat_func* my_unlinkat_func = NULL;
 static renameat_func* my_renameat_func = NULL;
 static futimesat_func* my_futimesat_func = NULL;
 static futimens_func* my_futimens_func = NULL;
+#ifndef _ALLBSD_SOURCE
```

Comment:
This change looks unrelated. Was it intentional?

Reply:
Yes. The variable on the next line `my_lutimes_func` caused an unused variable error in the macOS build.